### ISSUE TITLE
Syntax error even with latest version of nodejs

### ISSUE THREAD

**Issue by TeresaChou** [suan/vim-instant-markdown#136]
Hi, I might be asking a silly question but I just can't solve it.
This is my first time installing this plugin and it's not working. When I type instant-markdown-d, it shows syntax error:
```
/usr/local/lib/node_modules/instant-markdown-d/node_modules/debug/src/node.js:132
        let val = process.env[key];
        ^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/instant-markdown-d/node_modules/debug/src/index.js:9:19)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10) 
```
It looks like an issue of nodejs not supporting ES6, but I have updated my node to v10.11.0, and other node-modules work fine on my computer. Any help with this? I really hope it can work and I can enjoy this awesome plugin.

**Comment by junxxx** [comment]
Same problem on my Debian GNU/Linux 9.
    node -v  v8.12.0
    npm -v 6.4.1

**Comment by junxxx** [comment]
Hi.Buddy. I had the same problem few days ago.
Because the source code use node by using this:
    #!/usr/bin/node env
so, you should check you /usr/bin/node  -v to see what's the version.
I remove the system install nodejs by `apt-get remove nodejs` and link my latest version to /usr/bin/node
    ln -s path/to/you/latest/node/ /usr/bin/node

Good luck!

**Comment by TeresaChou** [comment]
Oh, now it actually launch the browser every time when I open something in vim, no matter what file type it is. Still looking for a way to solve it.

**Comment by junxxx** [comment]
> Oh, now it actually launch the browser every time when I open something in vim, no matter what file type it is. Still looking for a way to solve it.

add ` let g:instant_markdown_autostart = 0` to you .vimrc file

**Comment by Alopex4** [comment]
I get the syntax error too.
I use the follow command to solve it.
`sudo apt-get remove nodejs`
It was so weird....

**Comment by zhoujy** [comment]
> I get the syntax error too.
> I use the follow command to solve it.
> `sudo apt-get remove nodejs`
> It was so weird....

There is still nodejs in /usr/bin , it is old version ,so you should remove it

**Comment by Zoltan3057** [comment]
I met the same problem and I handle mine by:
1. download nodejs from https://nodejs.org/en/
2. link /usr/bin/node as said before:
`ln -s path/to/you/latest/node/ /usr/bin/node`
3. remove system installed nodejs： 
'apt-get remove nodejs'

**Comment by ashwinvis** [comment]
The version on github has solved this issue, it will be released as a bugfix version 0.1.1 soon (i.e. by @suan or the maintainers).
